src/components/com_tjcertificate/administrator/languages/en-GB/en-GB.com_tjcertificate.ini

```diff
@@ -383,3 +383,24 @@ COM_TJCERTIFICATE_PERMISSION_DELETE_EXTERNAL_CERTIFICATE="Delete training record
 COM_TJCERTIFICATE_PERMISSION_DELETE_EXTERNAL_CERTIFICATE_DESC="Allow users to delete training records"
 COM_TJCERTIFICATE_PERMISSION_DELETEOWN_EXTERNAL_CERTIFICATE="Delete own records"
 COM_TJCERTIFICATE_PERMISSION_DELETEOWN_EXTERNAL_CERTIFICATE_DESC="Allow users to delete own training records"
+
+;Bulk Record Config
+COM_TJCERTIFICATE_TJQUEUE_SETTING="Use TjQueue while add bulk records"
+COM_TJCERTIFICATE_TJQUEUE_SETTING_DESC="Set to 'Yes' if you want to use TjQueue while add bulk records (Note: To work this functionality you need com_tjqueue and certificate plugin enable)"
```

Comment:
Set to 'Yes' if you want to use TjQueue while adding bulk records (Note: To make it work, you need com_tjqueue component, and also you need to enable certificate queue plugin)

Reply:
updated